server/src/db/migrations/20210201211737-create-well.js

```diff
@@ -5,23 +5,19 @@ module.exports = {
       id: {
         allowNull: false,
         primaryKey: true,
-        type: Sequelize.STRING,
+        type: Sequelize.UUID,
```

Comment:
The bug ticket you are referring to is old and since then we have made a switch to have all of these as string

Reply:
Which is my bad for not being more on top of things